examples/partitionor_test_randomcircuits.py

```diff
@@ -0,0 +1,48 @@
+from qiskit import transpile

```

Comment:
Maybe we do not need to commit this file into this project. If necessary, it would be better to appear in our test directory